endogenous/scripts/end_transport.py

```diff
@@ -153,7 +211,7 @@ def transport(string):
         bus0="electricity bus",                               
         bus1 = "vehicle bus",
         carrier = "vehicle",
-        efficiency = 0.85,        
+        efficiency = bev_charge_efficiency,        
```

Comment:
The "EV" link should connect the "EV battery" with the "land transport demand" bus and its efficiency should be 1
see the reasoning in 
https://github.com/ricnogfer/pypsa-eur-sec_land_transport/issues/9

@BertoGBG I mentioned you here so that you realize this

Reply:
Okay, I would then use bev_charge_efficiency for the "battery charger" Link. Or would you keep the general efficiency for the battery inverter from the costs file?

Reply:
Yes, let's use the bev_charger_efficiency from the "battery charger" link and read it from the config file.